docs/docs/extra-features.md

```diff
@@ -55,23 +58,34 @@ module.exports = {
 
 `sku` comes with bundle analysis built in via [webpack-bundle-analyzer](https://www.npmjs.com/package/webpack-bundle-analyzer). A report is generated in the `/report` directory when `sku build` is run.
 
-## Pre-commit hooks
+## Pre-commit hook
 
-To speed up the feedback loop on linting and formatting errors, `sku` provides a `pre-commit` script that can be run to catch simple problems before CI. To make use of this, it's recommended that you install [husky](https://www.npmjs.com/package/husky) as a development dependency and configure it in `package.json` as follows:
+To speed up the feedback loop on linting and formatting errors, `sku` provides a `pre-commit` script that can be run to catch simple problems before CI.
+To make use of this hook, it's recommended to install [husky](https://www.npmjs.com/package/husky) as a development dependency and configure it as follows:
 
-```js
+```json
 // package.json
 
-"husky": {
-  "hooks": {
-    "pre-commit": "sku pre-commit"
+{
+  "scripts": {
+    "prepare": "husky install"
   }
-},
+}
+```
+
+```sh
```

Comment:
I think we should leave this out and just point to Husky's docs: https://typicode.github.io/husky/#usage. The way Husky manages hooks is an implementation detail and may change in the future.